app/js/views/reminders.jsx

```diff
@@ -104,6 +107,10 @@ export default class Reminders extends React.Component {
       });
   }
 
+  onParsingFailure() {
+    this.speechController.speak(`I didn't understand that. Can you repeat?`);
```

Comment:
Ah very good point!
We'll definitely need a ping/pong question and answers mechanism with preserved context for the lists. We can also use that when an information is missing from a reminder (Q: "Remind me of..." A: "When do you want me to remind you?"). When such a logic is in place we can totally improve this parsing failing scenario and have Cue to listen for a second phrase immediately.
I'd like to delay implementing this for now and take the time to build something solid.
How does it sound?